content/release-10-10-0/oee-10-10-0-bundle/10_10_0_3.md

```diff
@@ -0,0 +1,182 @@
+<td style="text-align:left">Formulas defining machine status and quality status conditions were not properly deleted. This led to problems when switching machine status and quality status calculation off. A similar problem occurred when enabling categories on a profile using machine and quality status calculations. This issue has been fixed, the formulas are now deleted correctly.</td>
+<td style="text-align:left">COE-543</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Using <i>Measurement count</i> in calculation rules was fixed to now start the calculation correctly.</td>
+<td style="text-align:left">COE-544</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The Andon board was fixed to include the default interval in the dropdown to switch between resolutions. This issue is related to COE-443.</td>
+<td style="text-align:left">COE-385</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The data chart on the machine dashboard was fixed to use correct UTC start date depending on the actual timezone. The wrong start date led to data gaps in the chart at the beginning of the selected timeframe. Also the data chart now includes the last measurement before the selected timeframe to ensure measurements are displayde for the entire selected timeframe without any gaps at the beginning of the selected timeframe.</td>
+<td style="text-align:left">COE-489<br/>COE-524</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">New Rest API endpoints have been added to allow deleting shift plans for a location and production plans for a device. See documentation at <a href="https://cumulocity.com/oee/api/10.10.0/">https://cumulocity.com/api/</a>.</td>
+<td style="text-align:left">COE-527</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Using calculation categories it is possible to define multiple formulas for the same input (for example, for availability loss to track different loss reasons). The results of the different formulas are aggregated to a total value. The calculation for categories has been fixed to not only store the total value, but also store and show the individual values used to calculate the total value.</td>
+<td style="text-align:left">COE-539</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Calculation for availability and quality losses was fixed when using status events to treat the conditions as amount and not as loss definitions.</td>
+<td style="text-align:left">COE-540</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">When adding machines to an organizational unit, it is now only possible to add machines with complete calculation profiles. Machines with incomplete calculation profiles are filtered from the list of options to select from.</td>
```

Comment:
"are filtered from the list of options to select from." -> "are filtered from the list of available options."

Reply:
@thomaswinkler @MarkJMReynolds-SAG Done